extensions/wikia/CloseMyAccount/CloseMyAccountSpecialController.class.php

```diff
@@ -174,6 +174,10 @@ public function reactivate() {
 			return;
 		}
 
+		// call to User::newFromConfirmationCode() is basically a token check
+		// tell CSRFDetector that we're fine here (PLATFORM-2206)
+		\Wikia\Security\CSRFDetector::onUserMatchEditToken();
```

Comment:
I'd create a separate method with better name even if it would call internally this one only. Worth changing?